JHUGenMELA/MELA/src/Mela.cc

```diff
@@ -809,6 +818,18 @@ void Mela::computeVHAngles(
   if (myVerbosity_>=TVar::DEBUG) MELAout << "Mela: End computeVHAngles" << endl;
 }
 
+//version without mVstar and mV, for backwards compatibility
```

Comment:
You need to remove this function. Not needed, nobody implemented VH angles to our knowledge because we never advertised it.